Automattic/wp-calypso
# Jetpack: Remove `<JetpackManageErrorPage />` in favor of `<EmptyContent />`.

#### Changes proposed in this Pull Request

A while ago, Jetpack had a module called "Manage". It was introduced to allow for remote management, which then allowed managing Jetpack sites remotely from WP.com and Calypso through XML-RPC and the REST APIs. As implementations grew, we had a `JetpackManageErrorPage` component that would be displayed in a variety of situations, including:

* When the Manage module is disabled
* When Jetpack is an old version that doesn't support Manage
* When the site is a Jetpack site (for domains, due to the lack of support for Jetpack sites)

Nowadays, only 2 cases remain, and one of them is not used, and the other one is used just once. Since the component is a simple wrapper around `EmptyContent`, this PR cleans it up in favor of using just `<EmptyContent />`.

#### Testing instructions

* Go to `/domains/:site` where `:site` is a Jetpack site.
* Verify the error screen looks the same as in production.
